packages/alert/src/index.vue

```diff
@@ -31,6 +31,20 @@
 <script lang='ts'>
 import { defineComponent, computed, ref, PropType } from 'vue'
 
+type AlertType = 'success' | 'info' | 'error' | 'warning';
+type AlertEffect = 'light' | 'dark';
+
+interface IAlertProps {
+  title: string
+  description: string
+  type: AlertType
+  closable: boolean
+  closeText: string
+  showIcon: boolean
+  center: boolean
+  effect: AlertEffect
+}
```

Comment:
Useless interface here

Reply:
I'm a little confused now 🤔, because some component files had the prop interface such as `packages/divider/src/index.vue` so i did this.